cmd/preflight/cli/run.go

```diff
@@ -129,6 +129,29 @@ func runPreflights(v *viper.Viper, arg string) error {
 		KubernetesRestConfig:   restConfig,
 	}
 
+	if v.GetString("since-time") != "" {
+		if v.GetString("since") != "" {
+			progressChan <- errors.Errorf("Only one of since-time / since may be used. The flag since-time will be used.")
```

Comment:
Is that how Kubectl handles it? Or does it abort because of invalid args?

Reply:
Yes I had my doubts about this too. K8s aborts with the message `error: parsing time`. I followed the error handling for the limits in the logs collector (communicating it through the progress channel), but it would make sense to abort because if someone used the flag and it can't be parsed maybe there would be no point in running through all the collectors and analyzers